src/Microsoft.ML.Core/Utilities/PathUtils.cs

```diff
@@ -67,13 +69,13 @@ public static string FindExistentFileOrNull(string fileName, string folderPrefix
             // 1. Search in customSearchDir.
             if (!string.IsNullOrWhiteSpace(customSearchDir)
                 && TryFindFile(fileName, folderPrefix, customSearchDir, out candidate))
-                    return candidate;
+                return candidate;
 
```

Comment:
Hmmm. I know you didn't write this code, but since the if condition is on multiple lines it really ought to be bracketed... if you have time could you fix it? The below if condition has the same problem I'm afraid.

Reply:
No worries. I'm always happy to clean up stuff.